### DISCUSSION TITLE
general feasability of extended time tracking plugin with external data integration?

### DISCUSSION THREAD

**Discussion by jdavisclark** [runelite/runelite#14995] [Q&A] [answered]
I've been kicking around the idea of writing a plugin to send time-tracker data to a configurable 3rd party (I'd probably write one, if people wanted to use it), to facilitate sending push notification to mobile-apps/smart-watches/etc...  The idea would be that you could configure getting push notifications for whatever farm/bird runs you want. It's something I've wanted for a while, so it seems like others might too. 

Generally speaking, would this kind of plugin be acceptable for inclusion in the official plugin hub? Does the answer to that change if the mobile apps required a $1/month subscription or something tiny to help cover backend service hosting cost?

Thanks!

**Comment by Nightfirecat** [comment, marked as answer]
Yes, that would be fine. Such a plugin would need to have a `warning` informing users that their data would be sent to a 3rd party source, or have default configurations such that no such connections were made, but I don't see any issue with this.

**Reply by jdavisclark** [reply]
excellent, thanks for the quick reply.

If I do it, the plan would be that the plugin doesn't even have a default configuration to send data anywhere, you'd have to configure it yourself based on whatever I ended up supporting (iftt, google sheets, custom webhook URL, etc...). So, there'd be a warning, and it'd be super explicit/obvious anyway.
